python-threatexchange/tests/test_api.py

```diff
@@ -30,3 +32,27 @@ def test_get_threat_privacy_groups_member(self):
         )
 
         self.assertTrue(isinstance(response[0], ThreatPrivacyGroup))
+
+    def test_get_threat_privacy_groups_owner(self):
+        """
+        Assumes that the app (if token is provided) will have at least one
+        privacy group.
+        """
+        response = self.api.get_threat_privacy_groups_owner()
+        self.assertTrue(
+            isinstance(response, collections.abc.Sequence)
+            and not isinstance(response, staticmethod),
```

Comment:
Why are we checking if its not a static method? I see its copied from above but I think we should remove